tests/fixtures/InMemoryEntityLookup.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Fixtures;
+
+use Wikibase\DataModel\Entity\EntityDocument;
+use Wikibase\DataModel\Entity\EntityId;
+use Wikibase\DataModel\Services\Lookup\EntityLookup;
+
+class InMemoryEntityLookup implements EntityLookup {
```

Comment:
*facepalm again*.....
This is why CR should not be the first thing you do in the day!